pkgs/by-name/ra/ranger/package.nix

```diff
@@ -12,18 +12,17 @@
   neoVimSupport ? true,
   improvedEncodingDetection ? true,
   rightToLeftTextSupport ? false,
-  gitUpdater,
+  unstableGitUpdater,
 }:
 
-python3Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication {
   pname = "ranger";
-  version = "1.9.4";
-
+  version = "1.9.3-unstable-2025-05-23";
```

Comment:
Why are we back at 1.9.3?

Reply:
Upstream's master branch is still on top of 1.9.3. 1.9.4 only exists on top of the `stable` branch. So while confusing, this is correct.